docs/components/concepts/job-workers.md

```diff
@@ -72,6 +72,11 @@ This could be useful when a job worker communicates with an external system. If
 This will result in an incident when the retries run out. Using the `retry back off` will delay the retry. This allows the external system some time to recover.
 If no `retry back off` the job is immediately retried.
 
+When `Completing or failing jobs` with variables, the variables are merged into the process of the job's associated task.
```

Comment:
When reading the Concepts documentation from start to finish, the Job Worker documentation comes before the Variables documentation. So readers are unlikely to be familiar with the Variables concept when they read this. 

🔧 I think we should link to it, so they can read more about Variables if they are interested.

```suggestion
When `Completing or failing jobs` with [variables](components/concepts/variables.md), the variables are merged into the process at the job's associated task.
```

Reply:
👍 Nice change here.